CromIAM/src/main/scala/cromiam/sam/SamClient.scala

```diff
@@ -93,11 +93,14 @@ class SamClient(scheme: String,
       userEnabled <- response.status match {
         case StatusCodes.OK =>
           val unmarshal: IO[UserStatusInfo] = IO.fromFuture(IO(Unmarshal(response.entity).to[UserStatusInfo]))
-          FailureResponseOrT.right[HttpResponse](unmarshal).map(info => info.enabled)
+          FailureResponseOrT.right[HttpResponse](unmarshal).map { info =>
+            if (!info.enabled) log.info("Access denied for user {}", user.userId)
+            info.enabled
```

Comment:
What's going on here?

Reply:
Oh, I see... `info` is a local variable, not a log level. Could we maybe rename it to `userInfo` to avoid similar future confusions?